src/main/resources/templates/layout/base.html

```diff
@@ -83,5 +103,27 @@
         window.GOVUKFrontend.initAll()
 
         </script>
+
+        <script>
+          function checkAndShowAppInfo(message){
```

Comment:
This is where the message is added to the page (if there is one).  It only makes one call when the page is loaded.